frontend/projects/setup-wizard/src/app/pages/success/success.page.ts

```diff
@@ -49,7 +49,7 @@ export class SuccessPage {
       const ret = await this.api.complete()
       if (!this.isKiosk) {
         this.torAddress = ret['tor-address']
-        this.lanAddress = ret['lan-address'].replace('https', 'http')
```

Comment:
do we actually want to leave this as is?

Reply:
I'm not actually sure what this is for

Reply:
oh, actually, I understand. Yes, we want to leave it, @MattDHill  still wants to encourage http immediately after setup on lan